lightwood/api/data_source.py

```diff
@@ -468,6 +504,10 @@ def get_decoded_column_data(self, column_name, encoded_data, decoder_instance=No
             decoded_data['predictions'] = preds
             decoded_data['class_distribution'] = pred_probs
             decoded_data['class_labels'] = labels
+        elif getattr(decoder_instance, 'normalize_by_group', False):
```

Comment:
Just check the class type and always pass it, the getattr thing makes it kinda too arcane. Even if we keep `normalize_by_group` we can still pass the group info all the time.